src/fmu/tools/rms/rename_rms_scripts.py

```diff
@@ -0,0 +1,520 @@
+"""Fix RMS Python script file extensions and gather useful information"""
+import argparse
+import logging
+import os
+import shutil
+from pathlib import Path
+from typing import Dict, List, Tuple, Union
+
+_logger = logging.getLogger(__name__)
+
+
+_BEGIN_HEADER = "Begin GEOMATIC file header"
+_END_HEADER = "End GEOMATIC file header"
+_BEGIN_PARAM = "Begin parameter"
+_END_PARAM = "End parameter"
+
+
+def _get_parser() -> argparse.ArgumentParser:
+    """Set up a parser for the command line utility"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "path",
+        type=str,
+        help=("Path to the RMS project"),
+    )
+    parser.add_argument("-v", "--verbose", action="store_true", help="Be verbose")
+    parser.add_argument(
+        "-b",
+        "--backup",
+        action="store_true",
+        help="Make a backup of the pythoncomp/ directory before doing anything",
+    )
+    parser.add_argument(
+        "-u",
+        "--unused",
+        action="store_true",
+        help="Print out a list of Python scripts that tracked by RMS but do not"
+        " appear to be used in any workflow",
+    )
+    parser.add_argument(
+        "-p",
+        "--pep8",
+        action="store_true",
+        help="Print out a list of Python scripts with PEP8 non-compliant"
+        " instance names",
+    )
```

Comment:
Perhaps it should be clear when using these options, no files will be changed?

Reply:
Added a `-t` flag for test runs with a description that indicates files will only _not_ change when it's flagged